site/_posts/2017-02-06-ansible-and-api-and-stuff.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Last Week in ManageIQ: API & Ansible & Stuff"
+author: Daniel Berger
+date: 2017-02-05 19:00:14 -07:00
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Hi all! [Daniel Berger](https://github.com/djberg96) here from the Providers team. This is my first Last Week in ManageIQ post, where I try to distill the week's highlights of ManageIQ development into a nice, digestable format.
+
+## Featured
+
+This past week saw [52 PR's][PRs merged last week] merged in ManageIQ core.
+
+Our provider repositories were active as well, with [7 Amazon PR's][Amazon PRs merged], [2 Azure PR's][Azure PRs merged], and [1 VMWare PR][VMWare PRs merged], in addition to 36 PR's merged on the classic and service UI side.
+
+## Improved
+
+This past week saw several improvements for Ansible, including the addition of Ansible Tower provider mixin by [Nick](https://github.com/ManageIQ/manageiq/pull/13733), while Brandon added an [automation manager type](https://github.com/ManageIQ/manageiq/pull/13720) as well as [job templates](https://github.com/ManageIQ/manageiq/pull/13495). Last week also saw Marcel add [graph refresh](https://github.com/ManageIQ/manageiq/pull/13585) for Ansible.
+
+### Everything's better in bulk.
+
+Meanwhile, Tim and Jillian were busy making improvements on the API side. Among other PR's, Jillian added the ability to [tag unassignments in bulk](https://github.com/ManageIQ/manageiq/pull/13712), as well as the ability to [tag multiple services in bulk](https://github.com/ManageIQ/manageiq/pull/13698). Meanwhile, Tim added [bulk delete for snapshots](https://github.com/ManageIQ/manageiq/pull/13711) and [nicer messages](https://github.com/ManageIQ/manageiq/pull/13710).
```

Comment:
Wondering if this should say "unassign tags in bulk" and/or "assign tags in bulk" - as Jillian added bulk stuff to all the things :tm: